Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml

```
@@ -3088,6 +3088,146 @@ script:
     - contextPath: Core.DynamicAnalysis.osSpawnerSigner
       description: ''
       type: String
+  - arguments:
+    - default: false
+      description: The sha256 of a file.
+      isArray: true
+      name: sha256
+      required: true
+      secret: false
+    deprecated: false
+    description: Get the prevalence of a file, identified by sha256.
+    execution: false
+    hidden: false
+    name: core-get-hash-analytics-prevalence
+    outputs:
+    - contextPath: Core.AnalyticsPrevalence.Hash.value
+      description: Whether the hash is prevalent or not.
+      type: Boolean
+    - contextPath: Core.AnalyticsPrevalence.Hash.data.global_prevalence
+      description: The global prevalence of the hash.
+      type: Unknown
+    - contextPath: Core.AnalyticsPrevalence.Hash.data.local_prevalence
+      description: The local prevalence of the hash.
+      type: Unknown
+  - arguments:
+    - default: false
+      description: The IP address.
+      isArray: true
+      name: ip_address
+      predefined:
+      - ''
+      required: true
+      secret: false
+    deprecated: false
+    description: Get the prevalence of an ip, identified by ip_address.
+    execution: false
+    hidden: false
+    name: core-get-IP-analytics-prevalence
```

commands should be all lower-case.
```suggestion
    name: core-get-ip-analytics-prevalence
```